### ISSUE TITLE
python modules installation fails

### ISSUE THREAD

**Issue by tgd-git** [lecomte/glasses-removal-gan#4]
hi i tried to follow the installation instructuions but getting errors on various modules.
(python3 -m pip install -r requirements.txt)

trouble with:
h5py==2.8.0
tensorflow==1.12.0
later on setup of werkzeug it stops at grpcio

i am on windows 10

**Comment by 35zhangxiaona** [comment]
I think maybe the author's enviroment is on Linux

**Comment by lecomte** [comment]
Yes, that is correct! I used a Linux based OS to develop this project, so I
am not quite sure on how to help with how to install those packages on
Windows.

Em sex., 3 de abr. de 2020 às 07:07, 35zhangxiaona <notifications@github.com>
escreveu:
